homeassistant/components/sensor/jewish_calendar.py

```diff
@@ -107,8 +108,18 @@ def state(self):
         """Update the state of the sensor."""
         import hdate
 
-        today = dt_util.now().date()
+        now = dt_util.as_local(dt_util.now())
+        _LOGGER.debug("Now: %s Timezone = %s", now, now.tzinfo)
+
+        today = now.date()
         upcoming_saturday = today + timedelta((12 - today.weekday()) % 7)
+        sunset = dt_util.as_local(get_astral_event_next(
```

Comment:
I am not sure why you would use start of local day, and then add 24 hours if it's before now? Why not just pass `now` to `event_next`?